### PR TITLE
Issue166-Adding unit test for ApplicationCache

### PR DESCRIPTION
**Pull request by juniartisu**

### CODE REVIEW

File: src/test/java/application/rest/v1/ApplicationCacheTest.java [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+    /**
+     * @throws java.lang.Exception
+     */
+    @SuppressWarnings("static-access")
+    @After
+    public void tearDown() throws Exception {
+    }
+
+    @Test
+    public void createApplicatigetNamespacedApplicationObject_succeeds() throws Exception {
```
Comment (rlcundiff) [inline src/test/java/application/rest/v1/ApplicationCacheTest.java]: Shouldn't the name just be "getNamespacedApplicationObject_succeeds"?

File: src/test/java/application/rest/v1/ApplicationCacheTest.java [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+            {
+                oneOf(coa).setApiClient(with(any(ApiClient.class)));
+                oneOf(coa).getNamespacedCustomObject("app.k8s.io", "v1beta1", "book-info", "applications", "book-info");
+                will(returnValue(jsonObject1));
+            }
+        });
+
+        try {
+            JsonObject result = ApplicationCache.getNamespacedApplicationObject(ac, "book-info", "book-info");
+            assertEquals("Test createApplicatigetNamespacedApplicationObject_succeeds FAILED", jsonObject1, result);
```
Comment (rlcundiff) [inline src/test/java/application/rest/v1/ApplicationCacheTest.java]: method name change follows through here and below.

**Review by rlcundiff — APPROVED** [review]
Please make the method name change and then merge.  We'll circle back to this one later to deal with the static methods.

File: src/test/java/application/rest/v1/ApplicationCacheTest.java (hunk 1 above)
Comment (juniartisu) [reply src/test/java/application/rest/v1/ApplicationCacheTest.java]: yap, thank you.

**Pull request merged**